jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/resources/org/jbpm/workbench/es/client/i18n/Constants.properties

```diff
@@ -105,4 +100,21 @@ UnacknowledgedErrors=Unackowledged Execution Errors
 AcknowledgedErrors=Acknowledged Execution Errors
 ViewProcessInstance=View process
 ViewTask=View task
-RequestRequeued=Request {0} put back into execution queue
\ No newline at end of file
+RequestRequeued=Request {0} put back into execution queue
+Cancel_Job=Cancel job
+Cancel_Job_Sure=Are you sure you want to cancel this Job?
+Cancel_Job_Confirmation=Cancel job confirmation
+Requeue_Job=Requeue job
+Requeue_Job_Sure=Are you sure you want to requeue this Job?
+Requeue_Job_Confirmation=Requeue job confirmation
+Bulk_Cancel_jobs=Bulk Cancel
+Cancel_Jobs=Cancel jobs
+Cancel_Jobs_Sure=Are you sure you want to cancel this Jobs?
```

Comment:
I feel bad to comment again, but could you please just change in the case of bulk actions the message from ```this Jobs``` to ```these Jobs```? :)

Reply:
@bsremac ups! my fault. Thanks for the comment!